src/Workspaces/Core/Desktop/Workspace/Host/TemporaryStorage/TemporaryStorageServiceFactory.MemoryMappedInfo.cs

```diff
@@ -346,19 +319,6 @@ public IntPtr GetPointer()
                 {
                     return (IntPtr)_start;
                 }
-
-                /// <summary>
-                /// Acquire the fixed pointer to the start of the memory mapped view.
-                /// The pointer will be released during <see cref="Dispose(bool)"/>
-                /// </summary>
-                /// <returns>The pointer to the start of the memory mapped view. The pointer is valid, and remains fixed for the lifetime of this object.</returns>
-                private static byte* AcquirePointer(MemoryMappedViewAccessor accessor)
```

Comment:
Interesting. It's always 0 in the current code but I'll account for it since we're planning for it to make a difference in the future.